inlong-sort-enhance/README.md

```diff
@@ -0,0 +1,50 @@
+
```

Comment:
how about using ` inlong-sort-sql` instead of `inlong-sort-enhance`, and `inlong-sort-enhance` be a submodule of `inlong-sort`.